apps/docs/src/routes/docs/theming/style-config-api.mdx

```diff
@@ -145,7 +145,7 @@ export const useAlertStyleConfig = createStyleConfig<AlertParts, AlertVariants>(
 
 ### Creating props type
 
-We can generate a _props type_ from the style config using the `StyleConfigProps` utility type.
+Using the `StyleConfigProps`` utility type, we can generate a _props type_ from the style config.
```

Comment:
Remove the second ` at the end of StyleConfigProps.

Reply:
Changed this as well